screeps-game-api/src/constants.rs

```diff
@@ -16,6 +16,7 @@
 
 use log::error;
 use num_derive::FromPrimitive;
+use num_traits::FromPrimitive;
```

Comment:
Isn't that shadowing the previous import?

Reply:
One's the derive and the other's the actual trait for usage - I think both are necessary?

Reply:
TIL that macros and items are not in the same namespace. Interesting!